schemas/records.json

```diff
@@ -267,25 +272,30 @@
             }
           ]
         }
-      ]
+      ],
+      "options": {
+        "hidden": true
+      }
     },
     "conformsTo": {
       "type": "array",
       "description": "The extensions/conformance classes used in this record.",
       "items": {
         "type": "string"
+      },
+      "options": {
+        "hidden": true
```

Comment:
This must be filled with specific conformance classes. How is this ensured?

Reply:
What are the options here? Where does the user get these conformance classes? Can we provide an enum?

Reply:
I guess the tooling should probably try to autodetect it as much as possible, but at least give users a chance to modify it.

Reply:
Ok, I unhid it and added some title, not sure yet how to fill this, but a problem for the future... https://github.com/ESA-EarthCODE/open-science-catalog-validation/pull/30/commits/ed67e5906b510c5e39eb5adb929f061cf25b8911